runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5221,12 +5221,17 @@ class INTERPRETER_CLASS
 				UDATA result = preparePinnedVirtualThreadForUnmount(_currentThread, object, true);
 				VMStructHasBeenUpdated(REGISTER_ARGS);
 				if (J9_OBJECT_MONITOR_OOM != result) {
+					PORT_ACCESS_FROM_JAVAVM(_vm);
 					restoreInternalNativeStackFrame(REGISTER_ARGS);
 					/* Handle the virtual thread Object.wait call.
 					 * VirtualThread.timeout is a private field used by both VM and JCL to temporarily hold
 					 * the value of expected wait/park time before a wake up task is scheduled using the value.
 					 */
 					J9VMJAVALANGVIRTUALTHREAD_SET_TIMEOUT(_currentThread, _currentThread->threadObject, millis + (nanos / 1000000));
```

Comment:
Suggest a local for `millis + (nanos / 1000000)`. Should it round up to the next millisecond?